test/DynamoCoreTests/CodeBlockNodeTests.cs

```diff
@@ -1280,6 +1339,7 @@ public void TestImperativePropertyAccessing()
         }
 
         [Test, Category("Failure")]
+        [Ignore("Test Loops Forever. Danger.")]
```

Comment:
@aparajit-pratap I added ignore to this test so that running dynamoCore tests locally is possible.